pkg/gitops/flux/installer.go

```diff
@@ -203,17 +207,28 @@ func (fi *Installer) Run(ctx context.Context) error {
 	logger.Info("Flux started successfully")
 
 	logger.Info("Committing and pushing manifests to %s", fi.opts.GitURL)
-	if err := fi.addFilesToRepo(ctx, cloneDir); err != nil {
+	if err := fi.addFilesToRepo(ctx, fi.clonedDir); err != nil {
 		return err
 	}
-	cleanCloneDir = true
 
 	logger.Info("Flux will only operate properly once it has write-access to the Git repository")
 	logger.Info("please configure %s so that the following Flux SSH public key has write access to it\n%s",
 		fi.opts.GitURL, fluxSSHKey.Key)
 	return nil
 }
 
+// DeleteCloneDir deletes the directory used for cloning the user's repository
+func (fi *Installer) DeleteCloneDir() {
```

Comment:
I think that the since `gitops apply` also clones a repository, we should decouple the git repository actions from either of them. I don't think it's a good design to have `gitops apply` piggybacking on `install flux`'s repository

Reply:
I think you are right. I'll make gitops apply get it's own clone of the repo so it doesn't rely on an implementation detali.

Reply:
Agree. I left flux installer as it was before, making gitops apply create its own clone. We can decide later if we extract all git operations outside the flux installer if we want.